samples/sample17/hello_world_oc.bal

```diff
@@ -21,15 +21,15 @@ import ballerina/openshift;
 
 @openshift:Route {
     host: {
-        domain: "<MINISHIFT_IP>.nip.io"
+        domain: "192.168.99.131.nip.io"
```

Comment:
We need to revert this. Its not suppose to change. I will check this separately.